ccd-definition/ComplexTypes/TestComplexType.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
why the empty file?

Reply:
don't know how this got here 😆 oops. will delete